README.md

```diff
@@ -66,3 +66,8 @@ Other JDBC Drivers can be added to the pom.xml file in the following section:
   </dependency>
 </dependencies>
 ```
+
+
+## License
+This repository is licensed under [MIT](/LICENSE) but components used in the built Docker images are under other licenses.
```

Comment:
Perhaps should state  `but components used in the Dockerfile examples are under other licenses`?

Reply:
Since we don't provide any built images